### PR TITLE
The science department now gets a mood bonus for petting borgs.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus
 add: The science department now gets a mood bonus for petting borgs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/silicon_defense.dm [hunk 1]
```diff
@@ -72,6 +72,8 @@
 		if ("help")
 			M.visible_message("<span class='notice'>[M] pets [src].</span>", \
 							"<span class='notice'>You pet [src].</span>")
+			if(M.job in list("Research Director", "Scientist", "Roboticist")) //Only true men know taste
+				SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "pet_borg", /datum/mood_event/pet_borg)
```
Comment (optimumtact) [inline code/modules/mob/living/silicon/silicon_defense.dm]: seems to be you should have a COMSIG_ADD_MOOD_EVENT_BORG_PET or something and only have specific jobs listen for it

Comment (Qustinnus) [reply code/modules/mob/living/silicon/silicon_defense.dm]: done, thanks for the suggestion.

**Comment by Fikou** [comment]
power crepe

**Comment by bgobandit** [comment]
fetish content

**Comment by Arathianen** [comment]
New powegamer meta: petting borgies :333

**Comment by zombiegums** [comment]
good pr

**Pull request merged**